bower_components/katex/dist/images/Image-Licensing-and-Technical-Notes.txt

```diff
@@ -0,0 +1,70 @@
+Copyright and Licensing
```

Comment:
Maybe it should be called `LICENSE.txt`?  I see, this isn't related to this PR at all, sorry!